AR-Top-MobileApp/Assets/Prefabs/MapController.cs

```diff
@@ -141,7 +139,7 @@ void buildLayers ()
 	{
 	
 		mapLayer = new GameObject ("MapLayer");
-		mapLayer.AddComponent <LeanRotate> ();
+		mapLayer.AddComponent <LeanRotate1> ();
```

Comment:
Can we not think of a better name for this?

Reply:
Changed it to LeanRotateYAxis